CHANGELOG.md

```diff
@@ -2,6 +2,27 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [2.3.0](https://github.com/lukso-network/tools-lsp-factory/compare/v2.1.0...v2.3.0) (2022-05-30)
```

Comment:
it takes from 2.1.0 instead of 2.2.0

Reply:
Nice spot. I already manually removed the entries from 2.2.0 so its correct. But need to look into why it happened though - I think something to do with the squash merge somehow